pages/dkp/kaptain/2.0.0/release-notes/2.0.0/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for Kaptain 2.0.0
+title: Release Notes for Kaptain 2.0.0
+menuWeight: 5
+beta: false
+excerpt: View release-specific information for Kaptain
+---
+
+**D2iQ&reg; Kaptain&reg; version 2.0.0 was released on April 25, 2022.**
+
+To get started with Kaptain, [download](../../download) and [install](../../install) the latest version of Kaptain.
+
+# Release Summary
+
+This release provides new features and capabilities that improve user experience, resolves reported issues, and changes the way Kaptain is deployed. This release supports Kubeflow 1.5 and a new version of the Kaptain SDK, 1.1.0. For more information on SDK 1.1.0, see [SDK release notes](../../sdk/1.1.x/release-notes).
+
+## New features and capabilities
+
+### Added Kaptain as DKP Catalog application
+
+TBD
+
+### Extended Support for Amazon EKS, Azure EKS and DKP multi-cluster
+
+TBD
+
+### Helm based deployment
+
+TBD
+
+### Support for Kubernetes 1.22 and Kubeflow 1.5
+
+TBD
+
+### Missed something?
+
+TBD
+
+## Fixes and Improvements
+
+- Moved from KFServing to KServe
```

Comment:
There are some SDK related fixes, but I think that has its own release notes?

Reply:
any cops fixes usually go here as well